src/main/java/database/operations/CompanyRepository.java

```diff
@@ -0,0 +1,132 @@
+package database.operations;
+
+import database.data.Company;
+import database.data.JdbcConnectionUtil;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompanyRepository {
```

Comment:
Please create an interface. Write there construct and implement the contract in this class.